packages/template-adapter/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ucp-npm/template-adapter",
-  "version": "0.0.12-alpha",
+  "version": "0.0.13-alpha",
```

Comment:
Should we up this to 1.0.0?

Reply:
It's pretty much a fake package, so I'm not worried about it